### PR TITLE
Update readme 

### PR DESCRIPTION
**Pull request by jasonandjay**
**In case of https://github.com/bitcoinjs/bitcoinjs-lib/issues/2106**

**1. Documented Error:**
Added details about the Uncaught TypeError: (0 , fs_1.readFileSync) is not a function error with tiny-secp256k1 due to WASM issues in browsers.

**2. ECC Library Alternatives:**
Introduced @bitcoinjs-lib/tiny-secp256k1-asmjs and @bitcoinerlab/secp256k1 as alternatives for improved browser support.

### CODE REVIEW

**Comment by jasonandjay** [comment]
unexpected test failed of audit, please have a look @junderw

**Comment by junderw** [comment]
Add ` -x 1097496` to the audit script to ignore. This vulnerability will at worst crash when running the tests or eslint.

**Comment by junderw** [comment]
1. Install a clean node_modules from `npm ci` (from package-lock)
2. Run `npm ls braces`
3. It shows which dependencies use it.
4. Upgrade all those dependencies to latest.
5. Refresh node_modules by deleting and re-runing npm ci
6. Check again

Since it did not fix the audit, that means there is no fix out yet and/or the chain of dependencies hasn't upgraded yet.

We can't do anything, but check the content of the vulnerability and see if we can ignore it.

I think we can ignore it.

**Comment by junderw** [comment]
I think instead of modifying the audit script in package.json we should just modify the invocation in CI.

If some other user wants to run audit, we shouldn't mess with it.

... maybe `-l high` should also be moved from package.json to main_ci?

File: README.md [hunk 1]
```diff
@@ -102,6 +102,22 @@ Which you can then import as an ESM module:
 <script type="module">import "/scripts/bitcoinjs-lib.js"</script>
 ````
 
+**Using Taproot:**  
+When utilizing Taproot features with bitcoinjs-lib, you may need to include an additional ECC (Elliptic Curve Cryptography) library. The commonly used tiny-secp256k1 library, however, might lead to compatibility issues due to its reliance on WASM (WebAssembly).
```
Comment (junderw) [inline README.md:106]: Backticks around tiny-secp256k1 so it's monospaced and stands out.

**Comment by jasonandjay** [comment]
Which part needs to be optimized?

**Review by junderw — APPROVED** [review]

**Pull request merged**